kodilla-testing2/Task-30-2.sql

```diff
@@ -0,0 +1,29 @@
+DROP PROCEDURE IF EXISTS UpdateBestsellers;
+
+DELIMITER $$
+
+CREATE PROCEDURE UpdateBestsellers()
+BEGIN
+
+    DECLARE RENT_MONTH, BOOKS_READ INT;
+    DECLARE FINISHED INT DEFAULT 0;
+    DECLARE BOOKS CURSOR FOR SELECT BOOK_ID FROM RENTS;
+    DECLARE CONTINUE HANDLER FOR NOT FOUND SET FINISHED = 1;
+    OPEN BOOKS;
+
+    WHILE (FINISHED = 0) DO
+        FETCH BOOKS INTO RENT_MONTH;
```

Comment:
Nie podoba mi się nazwa `RENT_MONTH`, do której przypisujesz id książki. To mocno mylące.